tests/util/test_logger.py

```diff
@@ -21,13 +22,36 @@ def _csv_to_dict(csv_path: str) -> dict:
     return result
 
 
+def _json_to_dict(json_path: str) -> dict:
+    result = defaultdict(list)
+    all_line_dicts = []
+    with open(json_path, "r") as f:
+        for line in f.readlines():
```

Comment:
Could replace with list comprehension: `all_line_dicts = [json.loads(line) for line in f.readlines()]`.